Dockerfile

```diff
@@ -14,6 +14,7 @@ USER tester
 COPY test /test
 
 VOLUME /config /secrets /cache
+VOLUME /log
```

Comment:
Is this intentional? It does not seem to be used and if it would be, it needs to be documented.

Reply:
Hmm… I can mount volume without this entry in Dockerfile so it's on its way out